js_modules/dagster-ui/packages/ui-core/src/instigation/TickHistory.tsx

```diff
@@ -403,6 +410,8 @@ function TickRow({
     return [added, deleted];
   }, [tick?.dynamicPartitionsRequestResults]);
 
+  const isOldTick = isOldTickWithoutEndtimestamp(tick, index);
```

Comment:
It would be nice to use the same terminology for this - could we say "isStuckStarted = isStuckStartedWithoutEndTimestamp" and then `isStuckStarted={isStuckStarted}`?